src/Entity/OgRole.php

```diff
@@ -139,6 +141,39 @@ public function setGroupBundle($group_bundle) {
   }
 
   /**
+   * Returns the role type.
+   *
+   * @return string
+   *   The role type. One of OgRoleInterface::ROLE_TYPE_IMMUTABLE or
+   *   OgRoleInterface::ROLE_TYPE_STANDARD.
+   */
+  public function getRoleType() {
+    return $this->get('role_type');
+  }
+
+  /**
+   * Sets the role type.
+   *
+   * @param string $role_type
+   *   The role type to set. One of OgRoleInterface::ROLE_TYPE_IMMUTABLE or
+   *   OgRoleInterface::ROLE_TYPE_STANDARD.
+   *
+   * @return $this
+   *
+   * @throws \InvalidArgumentException
+   *   Thrown when an invalid role type is given.
+   */
+  public function setRoleType($role_type) {
+    if (!in_array($role_type, [
+      self::ROLE_TYPE_IMMUTABLE,
```

Comment:
It is more than only undeletable, maybe just call it what it is: `ROLE_TYPE_DEFAULT`?